src/physics/em/BetheBlochInteractor.hh

```diff
@@ -0,0 +1,73 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheBlochInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/Units.hh"
+#include "BetheBlochInteractorPointers.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Brief class description.
+ *
+ * This is a model for XXXX process. Additional description
+ *
+ * \note This performs the same sampling routine as in Geant4's
+ * XXXX class, as documented in section XXX of the Geant4 Physics
+ * Reference (release 10.6).
+ */
+class BetheBlochInteractor
```

Comment:
Our focus is on the electron/photon physics, yes, though muons are next on the list after that. I added BetheBloch to this MR because @vpascuzz asked for an additional model to start working on during the hackathon. You are correct that BetheHeitler is being added by him as well in a separate branch :)